tests/gemini_test.yaml

```diff
@@ -0,0 +1,59 @@
+test_duration: 10080
+n_db_nodes: 1
+n_loaders: 1
+n_monitor_nodes: 1
+user_prefix: 'gemini-test'
+failure_post_behavior: keep
+ip_ssh_connections: 'public'
+experimental: 'true'
+append_conf: "enable_keyspace_column_family_metrics: true"
+instance_provision: 'spot_low_price'
+# instance_provision can be one of the following: on_demand|spot_fleet|spot_low_price|spot_duration
+store_results_in_elasticsearch: False
+
+# gemini
+gemini_run: True
+# the below cmd runs about 3 hours
+gemini_cmd: "gemini -d -n 1000000 -c 20 -p 100 -m mixed -f"
+gemini_url: 'https://s3.amazonaws.com/scylla-gemini/Binaries/gemini-0.2'
+gemini_static_url: 'https://s3.amazonaws.com/scylla-gemini/Binaries/schema.json'
+
+reuse_cluster: False
+db_nodes_public_ip: []
+db_nodes_private_ip: []
+mixed_db_nodes_public_ip: []
+mixed_db_nodes_private_ip: []
+loaders_public_ip: []
+loaders_private_ip: []
+monitor_nodes_public_ip: []
+monitor_nodes_private_ip: []
+
+backends: !mux
+    aws: !mux
+        cluster_backend: 'aws'
+        instance_type_loader: 'c4.4xlarge'
+        instance_type_monitor: 't2.small'
+        user_credentials_path: '~/.ssh/scylla-test'
+        us_east_1:
+            region_name: 'us-east-1'
+            security_group_ids: 'sg-5e79983a'
+            subnet_id: 'subnet-c327759a'
+            ami_db_scylla_user: 'centos'
+            ami_id_db_scylla: 'ami-0e2ad213d8babc2da'
+            ami_id_loader: 'ami-0e2ad213d8babc2da'
+            ami_loader_user: 'centos'
+            ami_id_db_cassandra: 'ami-0e2ad213d8babc2da'
+            ami_db_cassandra_user: 'centos'
+            ami_id_monitor: 'ami-0e2ad213d8babc2da'
+            ami_monitor_user: 'centos'
+
+databases: !mux
+    scylla:
+        db_type: scylla
+        instance_type_db: 'i3.4xlarge'
+    mixed_scylla:
+        db_type: mixed_scylla
+        instance_type_db: 'i3.large'
+        instance_type_db_mixed: 'i3.8xlarge'
```

Comment:
for better understanding which instance is this, I would call it:
"instnace_type_db_test_oracle"

Reply:
It's "oracle" just for the gemini test, but what if we want to create 2 or more scylla clusters?

Reply:
It's "oracle" just for the gemini test, but what if we want to create 2 or more scylla clusters?

Reply:
It's "oracle" just for the gemini test, but what if we want to create 2 or more scylla clusters for another purpose?

Reply:
I don't like the "oracle" name, it's just for gemini, it's not so clear if we want to create multiple scylla clusters